sphinx/templates/quickstart/conf.py_t

```diff
@@ -116,35 +116,6 @@ html_static_path = ['{{ dot }}static']
 htmlhelp_basename = '{{ project_fn }}doc'
 
 
-# -- Options for LaTeX output ------------------------------------------------
-
-latex_elements = {
-    # The paper size ('letterpaper' or 'a4paper').
-    #
-    # 'papersize': 'letterpaper',
-
-    # The font size ('10pt', '11pt' or '12pt').
-    #
-    # 'pointsize': '10pt',
-
-    # Additional stuff for the LaTeX preamble.
-    #
-    # 'preamble': '',
-
-    # Latex figure (float) alignment
-    #
-    # 'figure_align': 'htbp',
-}
-
-# Grouping the document tree into LaTeX files. List of tuples
-# (source start file, target name, title,
-#  author, documentclass [howto, manual, or own class]).
-latex_documents = [
-    (master_doc, '{{ project_fn }}.tex', u'{{ project_doc_texescaped_str }}',
-     u'{{ author_texescaped_str }}', 'manual'),
```

Comment:
We still have these template variables in quickstart code.
We have to deprecate them in nearly future.

Reply:
I think it would  be better if there was a link to a documentation of latex configuration.

Reply:
We've already have a link for configurations.
https://github.com/sphinx-doc/sphinx/blob/d2c3d54bb9a46c5e7a0d14c3345e489d7592de9a/sphinx/templates/quickstart/conf.py_t#L7

Did you mean that we should give a help only for latex configuration especially?

Reply:
Yes. Guidance to the document has effects like DeprecationWarning for those who search the keyword "latex" in "conf.py", I think that we can reduce troublesome question mails/issues.

Reply:
I'll also remove settings for manpages, texinfo and others. If my understanding is right, many comments will be listed in new conf.py. Is your goal like this? Or, would you like to add comment only for LaTeX output?
```
# -- Options for LaTeX output ------------------------------------------------
# see http://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-latex-output

# -- Options for man pages output ------------------------------------------------
# see http://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-manual-page-output

# -- Options for Texinfo output ------------------------------------------------
# see http://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-texinfo-output
```

Reply:
I'd personally be against this. If we're cleaning this up then there's no reason to go add more noise. The documentation for each of those builders is very easy to find on http://www.sphinx-doc.org/en/master/config. I don't think we need to be even more specific. Just my 2c

Reply:
ping @shimizukawa . Any comments?

Reply:
The config documentation page is big. Regarding latex and latex_documents in particular I do think the conf.py must have at least one commented out use case. With this, 99% of people will be up an running quick, without it they will go to http://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-latex-output and a large percentage will get scared and uncomfortable about spending 20 minutes reading customization where perhaps they don't know what LaTeX is.

Reply:
The purpose of `latex_documents` is:

* To split a document to multiple PDF files
* To change a docclass of LaTeX output
* To give a special title and authors information to LaTeX output

I understand changing a docclass is a bit important for some users. But others are less (I think).


If it was important, `man_pages` and `texinfo_documents` are also important. Is this right?

Reply:
I don't have a vote but I personally agree with everything you just said there, @tk0miya. Like it or not, HTML is still the default builder chosen by most users. These are likely the only config options we really need.